masterfiles/check_download_matches_git.py

```diff
@@ -0,0 +1,20 @@
+# check that the downloadable files match the git files, mitigating a build system supply-chain attack
```

Comment:
This looks like a good case for a docstring. Also it's not just about supply chain attack, I'd go with a bit more generic description:

```suggestion
"""Check that the downloadable files match the git files

This can be used to monitor / detect if something has been changed, accidentally or maliciously.
"""
```